client/js/sensorsCtrl.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+nugNgApp.controller('sensorsCtrl', ['$scope', '$http', function($scope, $http) {
+
+  var init = function() {
+    // just stick some data on scope
+    $scope.now = moment().format("dddd, MMMM Do YYYY, h:mm:ss a");
+    $scope.startValue = 5;
+	$scope.endValue = 15;
+	
+	
+  };
+  
+  	//console.log('test single', $scope.test.single);
+
+
+
+
+ 
+  init();
+	
+
+
+}]);
+
+
+angular.module('nouislider', []).directive("slider", function() {
```

Comment:
Hmmm, this is strange because... it recreates the angular directive.  This part should be done for us.  
If you look at his usage instructions, you'll see that there is no mention of writing a directive